charts/opentelemetry-collector/templates/NOTES.txt

```diff
@@ -19,5 +19,9 @@
 {{ end }}
 
 {{- if not (eq (toString .Values.containerLogs) "<nil>") }}
-[WARNING] 'containerLogs' is deprecated.  Use 'presets.logsCollection' instead. See https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/UPGRADING.md#0231-to-0240 for instructions on how to migrate."
+[WARNING] 'containerLogs' is deprecated.  Use 'presets.logsCollection' instead. See https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/UPGRADING.md#0231-to-0240 for instructions on how to migrate.
+{{ end }}
+
+{{- if not (eq (toString .Values.presets.logsCollection) "<nil>") }}
+[WARNING] 'logsCollection' "k8s.container.name", "k8s.namespace.name", "k8s.pod.name", "k8s.container.restart_count", "k8s.pod.uid" log-level attributes are promoted to resource-level attributes. Next release will remove the log-level attributes. If you need log-level attributes, please configure log collection manually.
```

Comment:
We don't have a standard release cadence for the charts, so `Next Release` doesn't mean much.  In this situations we normally go by the Collector Release cycle.  In this case I think by 0.62.0 makes sense.

```suggestion
[WARNING] 'logsCollection' "k8s.container.name", "k8s.namespace.name", "k8s.pod.name", "k8s.container.restart_count", "k8s.pod.uid" log-level attributes are promoted to resource-level attributes. Once Collector version 0.62.0 is released we will remove the log-level attributes. If you need log-level attributes, please configure log collection manually.
```